terraform/modules/prometheus/templates/alert.rules.tmpl

```diff
@@ -0,0 +1,9 @@
+groups:
+  - name: example
+    rules:
+    - alert: HighCPU
+      expr: avg by (space) (cpu{space=~"teaching-vacancies-.+"}) > 0
```

Comment:
same as the other comment, can space name be generic in the template?

Reply:
done, the default is emtpy config now